File: GenericTesting/ADODataAccess/DatabaseAccess.vb
Public Module DatabaseAccess
  Public Function GetTesterDatabase() As String
    Return "data source=(local);initial catalog=Tester;Integrated Security=False;password=pa55word;user id=sqluser;Connect Timeout=40;"
  End Function

  Public Function GetCentralTestDatabase() As String
    Return "data source=APC-Dev;initial catalog=PSG_APC_CENTRAL;Integrated Security=False;password=pa55word;user id=sqluser;Connect Timeout=40;"
  End Function
End Module

File: GenericTesting/GenericTesting/DataAccess/DatabaseAccess.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GenericTesting.DataAccess
{
  public static class DatabaseAccess
  {
    public const string TesterDatabase = "data source=(local);initial catalog=Tester;Integrated Security=False;password=pa55word;user id=sqluser;Connect Timeout=40;";
    public const string CentralTestDatabase = "data source=DEV-ENTERPRISE;initial catalog=PSG_APC_CENTRAL;Integrated Security=False;password=pa55word;user id=sqluser;Connect Timeout=40;";
    public const string EnterpriseTestDatabase = "data source=DEV-ENTERPRISE;initial catalog=PSG_ENTERPRISE;Integrated Security=False;password=pa55word;user id=sqluser;Connect Timeout=40;";
  }
}
